src/main/java/InputParser.java

```diff
@@ -0,0 +1,83 @@
+public class InputParser {
+    protected String[] args;
+    protected static String[] validCommands = {"mark", "m", "unmark", "um", "todo", "t", "deadline", "d", "event", "e", "list", "ls", "bye", "exit", "quit", "q"};
```

Comment:
I believe this is over 120 characters, perhaps make this into two separate lines?